msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -37,6 +37,550 @@ The following table provides a list of APIs available across the Microsoft Teams
 
 To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab running in the meeting context and is added for the response payload.
 
+### Examples
+
+The following are the TeamsJS v2 responses for Get user context API based on meeting type, user type, and call type:
+
+* Meeting type
+
+   # [Channel meeting](#tab/channel-meeting)
+
+   The following is a JSON payload response for a channel meeting for in-tenant users:
+
+   ```json
+    {
+     "app": {
+      "locale": "en-us",
+      "sessionId": "ff47ec00-e6a7-4dc1-a6ae-f44110f50c94",
+      "theme": "default",
+      "iconPositionVertical": 0,
+      "osLocaleInfo": {
+       "platform": "windows",
+       "regionalFormat": "en-in",
+       "shortDate": "dd-MM-yyyy",
+       "longDate": "dd MMMM yyyy",
+       "shortTime": "HH:mm",
+       "longTime": "HH:mm:ss"
+      },
+      "parentMessageId": "1678109354022",
+      "userClickTime": 1678109521159,
+      "userFileOpenPreference": "inline",
+      "host": {
+       "name": "Teams",
+       "clientType": "desktop",
+       "sessionId": "c3c3c0a0-f7a1-b070-6b89-c8cd1f380042",
+       "ringId": "ring1"
+      },
+      "appLaunchId": "7346ae66-5cac-47f9-8a0d-1228dac474cb"
+     },
+     "page": {
+      "id": "Test",
+      "frameContext": "sidePanel",
+      "subPageId": "",
+      "isFullScreen": false,
+      "isMultiWindow": true,
+      "sourceOrigin": ""
+     },
+     "user": {
+      "id": "57efa5f3-273c-47e2-a871-4879e5d849cf",
+      "displayName": "",
+      "isCallingAllowed": undefined,
+      "isPSTNCallingAllowed": undefined,
+      "licenseType": "Unknown",
+      "loginHint": "v-prkamble@microsoft.com",
+      "userPrincipalName": "v-prkamble@microsoft.com",
+      "tenant": {
+       "id": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+       "teamsSku": "enterprise"
+      }
+     },
+     "channel": {
+      "id": "19:49683807ffce4318ad6d6d7a24dbde45@thread.tacv2",
+      "displayName": undefined,
+      "relativeUrl": undefined,
```

Comment:
Would it be more useful to actually provide sample data here? Like use sample data from the Contoso or Northwind Traders tenant? 
Might be better to call that these out as actual values instead of just undefined.

Or at the very least, if we expect a string value to be returned let's set it to `''`

Reply:
We have checked using Contoso Demo tenant users and getting below response for "Channel Meeting"
Please suggest.

{
   "app":{
      "locale":"en-us",
      "sessionId":"254f89d1-ed9e-4053-bce2-436ad7c3e8b9",
      "theme":"dark",
      "iconPositionVertical":0,
      "osLocaleInfo":"undefined",
      "parentMessageId":"1681988550516",
      "userClickTime":1681988663326,
      "userFileOpenPreference":"inline",
      "host":{
         "name":"Teams",
         "clientType":"web",
         "sessionId":"62e054f0-8627-08aa-ad75-f2c428a764d3",
         "ringId":"general"
      },
      "appLaunchId":"6067819d-bf76-454b-95db-ae93ad9cdbe6"
   },
   "page":{
      "id":"Test",
      "frameContext":"sidePanel",
      "subPageId":"",
      "isFullScreen":false,
      "isMultiWindow":false,
      "sourceOrigin":""
   },
   "user":{
      "id":"4ebcc4d0-291b-4154-a85f-a89cd77aefa8",
      "displayName":"",
      "isCallingAllowed":"undefined",
      "isPSTNCallingAllowed":"undefined",
      "licenseType":"Unknown",
      "loginHint":"admin@M365x654992.onmicrosoft.com",
      "userPrincipalName":"admin@M365x654992.onmicrosoft.com",
      "tenant":{
         "id":"36a708ef-700d-4d60-9de0-0a5f7b7693df",
         "teamsSku":"unknown"
      }
   },
   "channel":{
      "id":"19:g_HQYwQPJ6UkbK6ZaObP3o2DbDHjIX2pXOyIDiPRbGM1@thread.tacv2",
      "displayName":"undefined",
      "relativeUrl":"undefined",
      "membershipType":"undefined",
      "defaultOneNoteSectionId":"undefined",
      "ownerGroupId":"undefined",
      "ownerTenantId":"undefined"
   },
   "chat":{
      "id":"19:g_HQYwQPJ6UkbK6ZaObP3o2DbDHjIX2pXOyIDiPRbGM1@thread.tacv2"
   },
   "meeting":{
      "id":"MCMxOTpnX0hRWXdRUEo2VWtiSzZaYU9iUDNvMkRiREhqSVgycFhPeUlEaVBSYkdNMUB0aHJlYWQudGFjdjIjMTY4MTk4ODU1MDUxNg=="
   },
   "sharepoint":"undefined",
   "team":{
      "internalId":"19:g_HQYwQPJ6UkbK6ZaObP3o2DbDHjIX2pXOyIDiPRbGM1@thread.tacv2",
      "displayName":"undefined",
      "type":"undefined",
      "groupId":"undefined",
      "templateId":"undefined",
      "isArchived":"undefined",
      "userRole":1
   },
   "sharePointSite":{
      "teamSiteUrl":"",
      "teamSiteDomain":"m365x654992.sharepoint.com",
      "teamSitePath":"",
      "teamSiteId":"",
      "mySitePath":"undefined",
      "mySiteDomain":"undefined"
   }
}

Reply:
@Pawan-MSFT thanks for checking. That response seems odd to me and makes me worried that the getContext call in a channel meeting sidepanel has a bug.

In particular, `channel`, `team` and `sharePointSite` being mostly undefined/empty when in a channel meeting is odd. I haven't tested a in-meeting sidepanel getContext, but in the pre channel meeting tab I got the following:

```
{
  "app": {
    "locale": "en-gb",
    "sessionId": "30bcd456-e934-4404-87ab-aa42cbbd0508",
    "theme": "default",
    "iconPositionVertical": 0,
    "parentMessageId": "1636424550912",
    "userClickTime": 0,
    "userFileOpenPreference": "inline",
    "host": {
      "name": "Teams",
      "clientType": "web",
      "sessionId": "3f0b2360-e94e-1a01-f96d-2eb379b87202",
      "ringId": "ring3_6"
    },
    "appLaunchId": "e747f2d2-c89f-44aa-b678-72799e222310"
  },
  "page": {
    "id": "test-entity-id",
    "frameContext": "content",
    "subPageId": "",
    "isFullScreen": false,
    "isMultiWindow": false,
    "sourceOrigin": ""
  },
  "user": {
    "id": "0c1e8f3e-6d5a-4611-86dc-cee5dc62af1e",
    "displayName": "",
    "licenseType": "Unknown",
    "loginHint": "<snipped>@<snipped>.onmicrosoft.com",
    "userPrincipalName": "<snipped>@<snipped>.onmicrosoft.com",
    "tenant": {
      "id": "77fb9fe4-<snipped>-<snipped>-<snipped>-92437fdf7162",
      "teamsSku": "unknown"
    }
  },
  "channel": {
    "id": "19:wfFCAo<snipped>_a18l1kta3G34aHtb01@thread.tacv2",
    "displayName": "channel_general",
    "relativeUrl": "/sites/<snipped>/Shared Documents/General",
    "membershipType": "Regular",
    "defaultOneNoteSectionId": ""
  },
  "chat": {
    "id": "19:wfFCAo<snipped>_a18l1kta3G34aHtb01@thread.tacv2;messageid=1636424550912"
  },
  "meeting": {
    "id": "MCMxOTp3ZkZDQW9<snipped>2ExOGwxa3RhM0czNGFIdGIwMUB0aHJlYWQudGFjdjIjMTYzNjQyNDU1MDkxMg=="
  },
  "team": {
    "internalId": "19:wfFCAo<snipped>_a18l1kta3G34aHtb01@thread.tacv2",
    "displayName": "<snipped>",
    "type": 0,
    "groupId": "4159b177-<snipped>-<snipped>-<snipped>-5f4932b7e17d",
    "isArchived": false,
    "userRole": 0
  },
  "sharePointSite": {
    "teamSiteUrl": "https://<snipped>.sharepoint.com/sites/<snipped>",
    "teamSiteDomain": "<snipped>.sharepoint.com",
    "teamSitePath": "/sites/<snipped>",
    "teamSiteId": ""
  }
}
```

Do you mind double checking with the engineering team that what you are seeing is expected. That values like `channel.displayName`, `channel.relativeUrl`, `team.displayName`, `team.groupId`, `sharePointSite.teamSiteUrl` and `sharePointSite.teamSitePath` should all be `"undefined"`?

Reply:
We will check with engineering team and will confirm